server-common/src/main/java/org/apache/kafka/pcoll/PHashMapWrapper.java

```diff
@@ -69,41 +48,4 @@ default <T> Map<K, T> asJava(Function<V, T> valueMapping) {
      * @return a wrapped persistent map that differs from this one in that the given mapping is removed (if necessary)
      */
     PHashMapWrapper<K, V> afterRemoving(K key);
```

Comment:
How about calling this `removed` and the other one `added`? That's the convention `scala` uses for non symbolic names and it seems to read a bit better than `after*`.